src/ontology/mondo-edit.obo

```diff
@@ -435483,6 +435483,26 @@ intersection_of: MONDO:0004691 ! autosomal dominant polycystic kidney disease
 intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/15672 ! ALG9
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700004
+name: idiopathic vs non-idiopathic
+is_a: MONDO:0021125 ! disease characteristic
+created_by: http://orcid.org/0000-0002-4142-7153
+
+[Term]
+id: MONDO:0700005
+name: idiopathic
+def: "A characteristic of a disease in which the disease has an uncertain or unknown cause." [https://github.com/monarch-initiative/mondo/issues/3177, https://orcid.org/0000-0002-4142-7153]
```

Comment:
For the definition, it is preferred to mention the parent class:
A disease characteristic in which....

(technically the parent is idiopathic vs non-idiopathic, but I think saying 'disease characteristic' works in this case)